### PR TITLE
chore: npmrc

### PR DESCRIPTION
**Pull request by breautek**
Progresses https://github.com/apache/cordova/issues/299

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,9 +18,9 @@
   "author": "Apache Software Foundation",
   "license": "Apache-2.0",
   "dependencies": {
-    "eslint": "^6.8.0",
+    "eslint": "^7.21.0",
```
Comment (erisu) [inline package.json:21]: Why was this changed?

File: package.json [hunk 2]
```diff
@@ -18,9 +18,9 @@
...
     "eslint-config-standard": "^14.1.1",
-    "eslint-plugin-import": "^2.20.1",
+    "eslint-plugin-import": "^2.22.1",
```
Comment (erisu) [inline package.json:23]: Why was this changed?

**Review by erisu — CHANGES_REQUESTED** [review]
I think these changes including the pacakge-lock.json change is incorrect.

**Comment by erisu** [comment]
Oh, it seems that this PR was based off another branch which also has its own PR.
I wonder if this PR should be based off master instead since the other PR is a major change and will require a major release. While this could go into a patch.

**Comment by breautek** [comment]
Yah I'll rebase. I didn't realise I branched off of another branch instead of master.

**Review by erisu — APPROVED** [review]
